### ISSUE TITLE
Not able to update to ionic cli 2.1.0

### ISSUE THREAD

**Issue by balasivagnanam** [driftyco/ionic-cli#1438]
Not able to use ionic rc0.

I have updated to ionic rc0, but the ionic create is working but when I serve the pages it gives 
Error: ENOENT: no such file or directory, open d:\ionic2app\www\index.html
issue

As per this thread https://github.com/driftyco/ionic/issues/8262

it is because ionic-cli is still in beta version mine is 2.0.0.beta-25.

I ran
```
npm uninstall ionic -g 
npm install ionic -g 
```
multiple times. but it is still showing the same version for me and not getting updated.
And I am not able to use anything because of this.

> npm WARN In ionic@2.1.0 replacing bundled version of semver with semver@4.2.0
> npm WARN In ionic@2.1.0 replacing bundled version of ionic-app-lib with ionic-app-lib@2.1.0-beta.1
> 
> C:\myapps\ionic2-tests\myApp>ionic --version
> 2.0.0-beta.25

> C:\myapps\ionic2-tests\myApp>npm version
> { npm: '3.10.6',
>   ares: '1.10.1-DEV',
>   http_parser: '2.7.0',
>   icu: '57.1',
>   modules: '48',
>   node: '6.2.1',
>   openssl: '1.0.2h',
>   uv: '1.9.1',
>   v8: '5.0.71.52',
>   zlib: '1.2.8' }

**Comment by jthoms1** [comment]
That is odd.  Try running the following instead.
```
npm uninstall -g ionic
npm install -g ionic@2.1.0
```

Please let me know if this works for you.

**Comment by balasivagnanam** [comment]
no it is still the same.. tried uninstall and install with version name in it.. but still it shows beta :-(

**Comment by balasivagnanam** [comment]
No effect its the same.

is there any logs or something that I can provide that might be helpful?

**Comment by balasivagnanam** [comment]
even after doing 
`npm uninstall ionic -g`

`ionic --version `
still it shows 2.0.0-beta25

**Comment by balasivagnanam** [comment]
though i cant see ionic in my npmlist output..

attaching npm list -g and npm list here..



[npmlist.txt](https://github.com/driftyco/ionic-cli/files/504324/npmlist.txt)
[npmlistall.txt](https://github.com/driftyco/ionic-cli/files/504325/npmlistall.txt)

**Comment by balasivagnanam** [comment]
>  "ionic": {
>       "version": "2.1.0",
>       "from": "ionic@2.1.0",
>       "resolved": "https://registry.npmjs.org/ionic/-/ionic-2.1.0.tgz",
>       "dependencies": {
>         "@ionic/app-generators": {
>           "version": "0.0.3",
>           "from": "@ionic/app-generators@0.0.3",
>           "resolved": "https://registry.npmjs.org/@ionic/app-generators/-/app-generators-0.0.3.tgz"
>         },
>         "cli-table": {
>           "version": "0.3.1",
>           "from": "cli-table@0.3.1",
>    

This is what I see inside npm list dump after installation.

but my ionic --version stil gives beta :-D

its been a bit funny..

but ionic serve still gives the same error.


[npmlistinstall.txt](https://github.com/driftyco/ionic-cli/files/504342/npmlistinstall.txt)

**Comment by balasivagnanam** [comment]
Finally I managed to solve it... uffff.

if you get ionic -v even after uninstalling..

run where ionic and find out the location of your installtion.

for me there was a version installed in
```
    C:\>where ionic
    C:\Program Files (x86)\nodejs\ionic
    C:\Program Files (x86)\nodejs\ionic.cmd
```

I am not sure how it went there last time. but after manually deleting them, My ionic cli is now updated and my project is able to render in browser.

Thanks a lot
